cypress/integration/userCanPaySubscriptions.spec.js

```diff
@@ -0,0 +1,107 @@
+describe('User can pay for subscription', () => {
```

Comment:
why is `userCanAccessPaymentForm.spec.js` necessary if you have this test?

Reply:
Then add the sad path to this file?

Reply:
That's actually a good question, its pretty redundant right? :P

Reply:
We removed the userCanAccessPaymentForm spec file and added the sadpath as extra it block to paysubscription test instead.